lib/workers/branch/index.ts

```diff
@@ -220,19 +220,6 @@ export async function processBranch(
       );
     }
 
-    if (
-      config.updateType !== 'lockFileMaintenance' &&
-      config.unpublishSafe &&
-      config.canBeUnpublished &&
-      (config.prCreation === 'not-pending' ||
-        /* istanbul ignore next */ config.prCreation === 'status-success')
-    ) {
-      logger.debug(
-        'Skipping branch creation due to unpublishSafe + status checks'
-      );
-      return ProcessBranchResult.Pending;
-    }
-
```

Comment:
This is the part I'm not sure about

Reply:
Should be possible to remove this as stability days will be added instead?